ReactWindows/ReactNative/Modules/DeviceInfo/DeviceInfoModule.cs

```diff
@@ -49,30 +55,80 @@ public override string Name
             }
         }
 
+        private static bool HasCoreWindow
+        {
+            get
+            {
+                return CoreApplication.MainView.CoreWindow != null;
+            }
+        }
+
         /// <summary>
         /// Called after the creation of a <see cref="IReactInstance"/>,
         /// </summary>
         public override void Initialize()
         {
             Context.AddLifecycleEventListener(this);
+            Context.AddBackgroundEventListener(this);
         }
 
         /// <summary>
         /// Called when the application is suspended.
         /// </summary>
         public void OnSuspend()
         {
-            ApplicationView.GetForCurrentView().VisibleBoundsChanged -= OnVisibleBoundsChanged;
-            DisplayInformation.GetForCurrentView().OrientationChanged -= OnOrientationChanged;
+            if (HasCoreWindow)
+            {
+                Unsubscribe();
+            }
         }
 
         /// <summary>
         /// Called when the application is resumed.
         /// </summary>
         public void OnResume()
         {
-            ApplicationView.GetForCurrentView().VisibleBoundsChanged += OnVisibleBoundsChanged;
-            DisplayInformation.GetForCurrentView().OrientationChanged += OnOrientationChanged;
+            if (HasCoreWindow)
+            {
+                Subscribe();
+            }
+        }
+
+        public void OnEnteredBackground()
+        {
+            if (HasCoreWindow)
+            {
+                Unsubscribe();
+            }
+        }
+
+        public void OnLeavingBackground()
+        {
+            if (HasCoreWindow)
+            {
+                Subscribe();
+            }
+        }
+
+        private void Subscribe()
+        {
+            if (!_isSubscribed)
+            {
+                _isSubscribed = true;
+                ApplicationView.GetForCurrentView().VisibleBoundsChanged += OnVisibleBoundsChanged;
+                DisplayInformation.GetForCurrentView().OrientationChanged += OnOrientationChanged;
+                SendUpdateDimensionsEvent();
```

Comment:
Check if DisplayInformation is available on background

Reply:
It is not.

---
In reply to: [145296660](https://github.com/Microsoft/react-native-windows/pull/1455#discussion_r145296660) [](ancestors = 145296660)